frameworks/kafka/universe/marathon.json.mustache

```diff
@@ -295,5 +295,14 @@
       "name": "api",
       "labels": { "VIP_0": "/api.{{service.name}}:80" }
     }
-  ]
+  ],
+  "check": {
+     "http": {
+        "portIndex": 0,
+        "path": "/v1/health"
+        },
```

Comment:
Closing `}` should be in the same column as `"http": {`

Reply:
Hi @samvantran , thanks for the review. I understand your comments and have acted upon them. I'll be careful next time to avoid such code formatting errs.